.github/workflows/coverage-report.yaml

```
@@ -0,0 +1,73 @@
+name: Code coverage report to Codecov
+
+on:
+  push:
+    branches: 
+      - master
+jobs:
+        
+  coverage:
+    runs-on: ubuntu-latest
+        
+    name: Create and upload coverage
+    steps:
+        
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+            
+      - name: Checkout code           
+        uses: actions/checkout@v2
+        
+      - name: Install Necessary Packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y net-tools libssl-dev gdb
+          sudo apt-get install gcovr
```

I think what @ihsandemir meant was this:
```suggestion
          sudo apt-get install -y net-tools libssl-dev gdb gcovr
```